### ISSUE TITLE
NorseBlue\ValueObjects\Immutable\ImmutableSimpleValueObject not found

### ISSUE THREAD

**Issue by peter279k** [norse-blue/php-scalar-objects#3]
**Describe the bug**

The completed error message is as follows:

```
PHP Fatal error:  Uncaught Error: Class 'NorseBlue\ValueObjects\Immutable\ImmutableSimpleValueObject' not found in /data/php-scalar-objects/src/PrimitiveType.php:13
```

**Environment**
Include as many relevant details about the environment you experienced the bug in and how to reproduce it.
- Versions used: `PHP 7.3.9`
- Operating system and version: `Ubuntu 18.04`

**To Reproduce**
1. clone this repository on master brnch
2. run `php7.3 composer.phar install`.
3. run `php7.3 vendor/bin/phpunit`
4. See fatal error message

**Expected behavior**
The unit tests are completed without any errors.

**Screenshots**
There is not screenshot at this moment.

**Additional context**
After looking at this namespace, this class belongs to the `norse-blue/value-objects` package.

And it seems that the `norse-blue/value-objects` class namespaces has been changed.

The `NorseBlue\ValueObjects\Immutable\ImmutableSimpleValueObject` is not existed on the [value-objects](https://github.com/norse-blue/php-value-objects) anymore.

That's also the reason why latest Travis CI build is failed.

The latest Travis CI build log is available [here](https://travis-ci.com/norse-blue/php-scalar-objects/builds/126641136).

**Comment by axelitus** [comment]
Hi @peter279k, I can confirm this is currently an issue due to the latest changes in the referred package.

I haven't had quite the time to tackle this yet, but it's on the roadmap to update the package and even tag a first stable release.

**Comment by axelitus** [comment]
This issue has been fixed in [Release 0.1.0](https://github.com/norse-blue/php-scalar-objects/releases/tag/0.1.0)
